tooling/strace_analysis.sh

```diff
@@ -169,15 +169,24 @@ filterStraceFiles() {
 processFiles() {
     echo "Processing found files to determine 'Package' versions... (this will take a few minutes)"
 
+    # Determine OS package query command
+    os_type=centos
+    package_query="rpm -qf"
+
+    if grep "Alpine Linux" /etc/os-release >/dev/null 2>&1; then
+        # Alpine
+        os_type=alpine
+        package_query="apk info --who-owns"
+    elif grep "Ubuntu" /etc/os-release >/dev/null 2>&1; then
```

Comment:
This should check for `debian` rather than Ubuntu to catch more valid distributions (This suggestion below may be overkill, but we should definitely avoid referencing Ubuntu directly)
```suggestion
    elif grep "^ID,*debian" /etc/os-release >/dev/null 2>&1; then
```

Reply:
(Note: I have not tested the above suggestion myself in the script)